index.html

```diff
@@ -36,6 +36,13 @@ <h2>Skills</h2>
             <ul>
             </ul>
         </section>
+
+        <section id="Projects">
+            <h2>Projects</h2> 
+            <ul>
+                <li><a href="https://github.com/Alarlar/dinar-ibragim-haumea">https://github.com/Alarlar/dinar-ibragim-haumea </a></li> 
```

Comment:
This shouldn't be hardcoded. You're going to add your projects dynamically using js.